test/acceptance/features/steps/steps.py

```diff
@@ -267,14 +268,8 @@ def then_envFrom_contains(context, app_name, sbr_name1, sbr_name2):
     time.sleep(60)
     openshift = Openshift()
     result = openshift.get_deployment_envFrom_info(app_name, context.namespace.name)
-
-    # Expected result from 'oc' (openshift client) v4.5
-    expected_result_oc_45 = f'[map[secretRef:map[name:{sbr_name1}]] map[secretRef:map[name:{sbr_name2}]]]'
-    # Expected result from 'oc' (openshift client) v4.6+
-    expected_result_oc_46 = f'[{{"secretRef":{{"name":"{sbr_name1}"}}}},{{"secretRef":{{"name":"{sbr_name2}"}}}}]'
-
-    assert result == expected_result_oc_45 or result == expected_result_oc_46, \
-        f'\n{app_name} deployment should contain secretRef: {sbr_name1} and {sbr_name2}\nActual secretRef: {result}'
+    result | should.be_equal_to("[map[secretRef:map[name:binding-request-1]] map[secretRef:map[name:binding-request-2]]]")\
+        .desc(f'{app_name} deployment should contain secretRef: {sbr_name1} and {sbr_name2}')
```

Comment:
why do we reverting support for oc 4.5 and 4.6?

Reply:
This change is uninted, reverting back. Thanks